spec/.github/ISSUE_TEMPLATE/01-compatible-certification-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Compatible Certification Request
+about: Request certification of a compatibile implementation of MicroProfile Platform release
+title: 'Compatibility certification request for [product/project name] for MciroProfile [plaftorm release verseion]'
+labels: 'Certification :trophy:'
+assignees: ''
+
+---
+
+- [ ] Organization Name ("Organization") and, if applicable, URL:<br/>
+  // Add here
+- [ ] Product Name, Version and download URL (if applicable):<br/>
+  // Add here
+- [ ] Specification Name, Version and download URL:<br/>
+   // Add here
+- [ ] (Optional) TCK Version, digital SHA-256 fingerprint and download URL:<br/>
```

Comment:
Do you mean all 8 TCKs?

Reply:
nope. This should be only any potential TCKs added by platform. At the moment, there is none. Maybe just remove this to reduce confusion. I think asking people to type in the url for the 8 tcks are not useful.

Reply:
But if you file a certification request for the platform, you need to pass the different spec TCKs.  So how does this need to be provided (evidence that you pass them)?

Reply:
@rdebusscher please see the following bulletin point. You need to list all of the tck results for the 8 specs ([here](https://openliberty.io/certifications/microprofile/4.0/TCKResults.html) is an example)